### PR TITLE
Promote Custom Threshold Rule to Beta

### PR DESCRIPTION
**Pull request by bmorelli25**
### Summary

Closes https://github.com/elastic/observability-docs/issues/3421

### CODE REVIEW

File: docs/en/observability/threshold-alert.asciidoc [hunk 1]
```diff
@@ -151,4 +151,4 @@ You can add more context to the message by clicking the icon above the message t
 and selecting from a list of available variables.
 
 [role="screenshot"]
-image::images/logs-threshold-alert-default-message.png[Default notification message for logs threshold rules with open "Add variable" popup listing available action variables,width=600]
\ No newline at end of file
+image::images/logs-threshold-alert-default-message.png[Default notification message for logs threshold rules with open "Add variable" popup listing available action variables,width=600]
```
Comment (mdbirnstiehl) [inline docs/en/observability/threshold-alert.asciidoc:154]: Looks like this screenshot still has Technical Preview in the heading.

image

Comment (mdbirnstiehl) [reply docs/en/observability/threshold-alert.asciidoc:154]: sorry, not the screenshot in line 154, but the one in line 13

Comment (bmorelli25) [reply docs/en/observability/threshold-alert.asciidoc:154]: good spot. done!

**Review by mdbirnstiehl — APPROVED** [review]
👌

**Pull request merged**